src/test/java/com/google/cloud/tools/maven/it/RunMojoIntegrationTest.java

```diff
@@ -86,6 +97,34 @@ public void run() {
     assertEquals("Hello from the App Engine Standard project.", urlContent.toString());
     verifier.verifyErrorFreeLog();
     verifier.verifyTextInLog("Dev App Server is now running");
+    verifier.verifyTextInLog("Module instance " + expectedModuleName + " is running");
+  }
+
+  @SuppressWarnings("unused") // used for testRun()
+  private Object[] parametersForTestRun() {
+    List<Object[]> result = new ArrayList<>();
+    for (SupportedDevServerVersion serverVersion : SupportedDevServerVersion.values()) {
+      for (String[] profileAndServiceName : new String[][]{
+        { "", "standard-project"},
+        { "base-it-profile,appyamls", "standard-project-appyamls"},
+        { "base-it-profile,services", "standard-project-services"}
+        } ) {
+        result.add(
```

Comment:
I agree, it's pretty hard to read what's going on. I think I prefer the first option for now.